src/NLog/Targets/DefaultJsonSerializer.cs

```diff
@@ -732,5 +733,18 @@ private static PropertyInfo[] GetPropertyInfosNoCache(Type type)
         }
 
 
+        #region Implementation of ISerializer
+
+        /// <summary>Serialize an object</summary>
+        /// <param name="sb">Add serialized value to this builder</param>
+        /// <param name="value">Value to be serialized</param>
+        /// <param name="formatProvider">format</param>
+        public void SerializeObject(StringBuilder sb, object value, IFormatProvider formatProvider)
+        {
+            //todo inject options somewhere else
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1135)